Release/include/cpprest/containerstream.h

```diff
@@ -113,7 +113,7 @@ class basic_container_buffer : public streams::details::streambuf_state_manager<
 
         msl::safeint3::SafeInt<size_t> readhead(m_current_position);
         msl::safeint3::SafeInt<size_t> writeend(m_data.size());
-        return (size_t)(writeend - readhead);
+        return (size_t)writeend - (size_t)readhead;
```

Comment:
This change breaks the whole purpose of using SafeInt (since you take the values out of the SafeInt before doing the math). I don't think we can accept that.

Reply:
ok.  I think the fundamental issue was the need to explicitly define the default copy constructor, which I fixed afterwards.